public/app/views/index.html

```diff
@@ -27,31 +28,36 @@
     <link rel="stylesheet" type="text/css" href="assets/css/style.css">
 </head>
 
-<body ng-app="kibibitCodeEditor" ng-controller="mainController as main" class="main-app-container" ng-class="{foo:true}">
-    <div ui-layout="{ dividerSize: '0' }"  >
-      <div ui-layout-container size="104px" class="menu-bar-container">
-        <menu-bar></menu-bar>
-      </div>
-      
-      <div ui-layout-container="central" > 
-        <div ui-layout="{ flow : 'column', dividerSize: '10' }">
-          <div ui-layout-container class="sidebar-container" size="20%">
-              <div class="sidebar" ng-scrollbars>
-                <kb-file-tree 
-                  kb-file-tree-path="main.projectFolderPath"
-                  kb-file-tree-selection="main.openFile">
-                </kb-file-tree>
+<body ng-app="kibibitCodeEditor">
+    <div ng-controller="mainController as main" class="main-app-container" ng-class="{'app-initiated':true}">
+        <div ui-layout="{ dividerSize: '0' }"  >
+          <div ui-layout-container size="104px" class="menu-bar-container">
+            <menu-bar></menu-bar>
+          </div>
+          
+          <div ui-layout-container="central" style="z-index: 6;"> 
+            <div ui-layout="{ flow : 'column', dividerSize: '10' }">
+              <div ui-layout-container class="sidebar-container" size="20%">
+                  <div class="sidebar" ng-scrollbars>
+                    <kb-file-tree 
+                      kb-file-tree-path="main.projectFolderPath"
+                      kb-file-tree-selection="main.openFile">
+                    </kb-file-tree>
+                </div>
+              </div>
+              <div ui-layout-container class="main-view-container">
+                <div class="tabs"></div>
+                <div class="main-view" ng-view></div>
+              </div>
             </div>
           </div>
-          <div ui-layout-container class="main-view-container">
-            <div class="tabs"></div>
-            <div class="main-view" ng-view></div>
+          <div ui-layout-container size="30px" class="status-bar">
+            <span>Line {{main.settings.cursor.row}} - Column {{main.settings.cursor.column}}</span>
           </div>
         </div>
-      </div>
-      <div ui-layout-container size="30px" class="status-bar">
-        <span>Line {{main.settings.cursor.row}} - Column {{main.settings.cursor.column}}</span>
-      </div>
+    </div>
+    <div class="loading-logo" style="position: absolute; top: 0; bottom: 0; left: 0; right: 0; z-index: 5; display: flex; align-items: center; justify-content: center;" ng-class="{'app-initiated':true}">
```

Comment:
Basically, we don't use style unless it's something that needs to be loaded fast. Here, we want the loading logo to load before everything else. This way, we make sure it is rendered as fast as possible.

In the `menuBarTemplate.html` I used it on the `paste` button since that button is disabled permanently (notice the `disabled="disabled"` attribute on the button, which isn't controlled by a parameter as well)

besides that, I used `ng-style` for now since the `md-icon` directive stop working if I add another class there. The solution later will be to change the color of the parent element instead. But that will go into a different issue that I plan to open about making disabled in the menuBar more structured